.github/workflows/lucode2-check.yaml

```diff
@@ -56,6 +56,11 @@ jobs:
 
       - uses: r-lib/actions/setup-pandoc@v1
 
+      - name: Set up Python 3.9
```

Comment:
I don't see any Python code added, why would you need to add a Python reference here? Was this added automatically by some RSE code? If yes, could you double check with them if this change is necessary.

Reply:
same as above - no idea why or how that was added. 
I put a question in the RSE channel

Reply:
I personally think you don't need to worry about changes like these, as long as you use the latest versions of libraries when running buildLibrary. They were introduced by running buildLibrary, so you can assume that RSE intended this.

Reply:
again, I would contact RSE to create a PR that includes only these changes and the justification why they were added.
IN your case I would simply edit those files by hand and remove the changes that are not related with your PR.